### ISSUE TITLE
Crash on compiling with 0.6.0-alpha.3

### ISSUE THREAD

**Issue by enugentdt** [PyO3/pyo3#366]
Hi, all

## 🐛 Bug Report

I admit, I am using pyo3-pack. However, running the `build` command causes an error within pyo3:

```bash
+ pyo3-pack build
🔗 Found pyo3 bindings
🐍 Found Python 2.7mu at python2.7, Python 3.5m at python3.5, Python 3.6m at python3.6, Python 3.7m at python3.7
error[E0433]: failed to resolve: could not find `ReturnTypeIntoPyResult` in `pyo3`
  --> src/lib.rs:20:1
   |
20 | #[pyfunction]
   | ^^^^^^^^^^^^^ could not find `ReturnTypeIntoPyResult` in `pyo3`

error[E0107]: wrong number of lifetime arguments: expected 0, found 1
  --> src/lib.rs:20:1
   |
20 | #[pyfunction]
   | ^^^^^^^^^^^^^ unexpected lifetime argument

error: aborting due to 2 previous errors

Some errors occurred: E0107, E0433.
For more information about an error, try `rustc --explain E0107`.
[==========================================================> ]  43/44  rust-markdown-text-puller

Cargo build finished with "exit code: 101": `cargo rustc --message-format json --quiet --manifest-path Cargo.toml --features pyo3/python2 --lib --`
Failed to build a native library through cargo
```

After some searching in the code, it appears `ReturnTypeIntoPyResult` was removed in 0.6.0-alpha.3, however, it _appears_ that it is still looking.

### 🌍 Environment

 - OS: Mac OS X & manylinux
 - Python: 2.7 & 3.6
 - Python install: Core install, OS X is from brew, my manylinux container has it from its upstream docker repo
 - Rust version: rustc 1.34.0-nightly (d17318011 2019-02-07)
 - PyO3 version: 0.6.0-alpha.3

### 💥 Reproducing

A minimum working example is the sample code. Put that in a `src/lib.rs`, make a Cargo.toml, then run `pyo3-pack build` on it. I've been able to replicate it on manylinux and OS X, and noticed that the offending type was missing from the 0.6.0-alpha.3 docs. In the meantime, I'm going to alpha.2 to make my prod builds work. Fwiw, I've attached my Cargo.toml, if only so it saves looking for somewhere to copy+paste one :P

```toml
[package]
name = "pyo3-0.6.0-alpha.3-error-demo"
version = "0.1.0"
authors = ["Human"]
edition = "2018"

[lib]
name = "pyo3_alpha_3_error_demo"
crate-type = ["cdylib"]

[dependencies]
pyo3 = { version = "0.6.0-alpha.2", features = ["extension-module"] }
```

**Comment by konstin** [comment]
Oh I pyo3 still had the wrong alpha for pyo3cls and pyo3-derive-backend pinned. Fixed in 0cd72ac213c66b135a6f03dfea14d5a3235ad95f.

**Comment by enugentdt** [comment]
Cool! Glad I could be of at least vague help.
